### ISSUE TITLE
Destination paths are broken on Windows

### ISSUE THREAD

**Issue by kthu** [vladh/lein-sassy#6]
On Windows, the entire src path is treated as a filename such that

```clojure
:sass {:src "src/scss/" 
       :dst "resources/public/css/"}
```

will produce a css file in `resources/public/css/src/scss/` and not in the expected `resources/public/css/`

**Comment by vladh** [comment]
Sorry! I'll look into this.

**Comment by denysbutenko** [comment]
:+1:

**Comment by vladh** [comment]
I'd really like to fix this — it's a bit difficult to do since I don't have a Windows environment handy. If anyone knows what the issue is in the code by the time I get a Windows environment that would help. Thanks guys.

**Comment by zhanghuabin** [comment]
Hi, @vladh, I guess the root cause should be the File.separtor is "\" on windows but we use "/" in src setting.

Please see the following line in renderer.clj:

`insubpath (s/replace-first inpath (:src options) "")`

We say the settings is `{:src "src/scss/"}`, the `inpath` will be like `src\scss\abc.scss` on the win platform, but the `insubpath` won't be the expected `abc.scss`.
Eventually, the `outpath` will be like `resources/public/css/src\scss\abc.scss`.

**Comment by vladh** [comment]
@zhanghuabin Brilliant! Thank you for the PR! Just to confirm, you've tested that this fixes the problem on Windows?

**Comment by zhanghuabin** [comment]
@vladh Yep,I did two tests on win8:

- Before the fix, I use lein-sassy 1.0.7, and the settings is

`:sass {:src "resources\\app\\scss\\" :dst "resources/public/css/"}`

Run `lein sass once` and get the expected result.

It shows my analysis about root cause is right.

-  I test the following code snippets in repl:

```
(def files (file-seq (clojure.java.io/file (System/getProperty "java.io.tmpdir"))))

(doseq [file files]
  (let [options {:src "resources/public/css/"}
        inpath (.getPath file)
        insubpath (clojure.string/replace-first (.replaceAll inpath "\\\\" "/") (.replaceAll (:src options) "\\\\" "/") "")]
    (println insubpath)))
```

I know the fix looks not graceful, but I think which is good enough for most Win users, at least for me :smile:

**Comment by vladh** [comment]
Oh no, it's fine, thank you so much for fixing it!
